src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Startup.cs

```diff
@@ -79,8 +79,8 @@ public void ConfigureServices(IServiceCollection services)
         // cause the data source builder to throw a host exception.
         if (!_hostEnvironment.IsIntegrationTest())
         {
-            var dataSourceBuilder = new NpgsqlDataSourceBuilder(_configuration.GetConnectionString("PublicDataDb"));
-            dataSourceBuilder.MapEnum<GeographicLevel>();
```

Comment:
After removing this line, I'm now seeing this again in the DB. Are we sure we don't need this line? 
![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/148866614/f8086b31-41c7-4172-96a7-5f79b0e9eebf)